### PR TITLE
Add section how to indicate specific modification timestamp

### PR DESCRIPTION
**Pull request by holtkamp**

### CODE REVIEW

File: docs/adapter/phpcr.md [hunk 1]
```diff
@@ -49,3 +49,16 @@ $session = $repository->login();
...
+
+```php
+$path = '/path/to/file.ext';
+$content = file_get_contents($path);
+$config = ['timestamp' => filemtime($path)]; //Use the time when the content of the file was last changed.
+
+$filesystem->write($path, $content, $config);
+```
+
+This can be usefull when the file timestamp needs to be preserved when copying a file structure to PHPCR.
```
Comment (dbu) [inline docs/adapter/phpcr.md]: s/usefull/useful/

File: docs/adapter/phpcr.md [hunk 2]
```diff
@@ -49,3 +49,16 @@ $session = $repository->login();
 $root = '/flysystem_tests';
 $filesystem = new Filesystem(new PhpcrAdapter($session, $root));
 ```
+
+### Indicate specific modification timestamp when writing content
+By default PHPCR will use the "current" timestamp as the "last modified" timestamp of an entry when writing content. A specific timestamp can be provided by using the configuration array:
```
Comment (dbu) [inline docs/adapter/phpcr.md]: can we say `use the current system time as the "last modified" timestamp`?

File: docs/adapter/phpcr.md (hunk 1 above)
Comment (holtkamp) [reply docs/adapter/phpcr.md]: https://whichiscorrect.com/useful-vs-usefull/ oops!

Comment (dbu) [reply docs/adapter/phpcr.md]: i checked as well, to be sure :-)

**Pull request merged**